source/releasenotes/2024-03-26-terminus-github-action-sessions-cache.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Terminus Github Action Session Caching
+published_date: "2024-03-26"
+categories: [new-feature, tools-apis]
+---
+We've updated our [Terminus GitHub Action](https://github.com/marketplace/actions/setup-terminus) with the ability to share sessions across jobs to simplify and reduce the authentication overhead including any associated rate limiting when authenticating their machine token. These sessions are encrypted prior to being cached using industry standard AES-256 encryption, PBKDF2 (Password-Based Key Derivation Function 2) algorithm, and increased iterations when deriving the encryption key. We've also updated to the latest versions of `actions/checkout` and `actions/cache`.
+
+Some technical details:
+
+- Session is encrypted using OpenSSL with AES-256 and PBKDF2 before being cached.
+
+- Session is scoped to workflow run, meaning re-running same workflow will use existing cache (in case of a failure, etc) while new workflows create new session.
```

Comment:
```suggestion
- The session information is scoped to a workflow run, meaning re-running the same workflow will use existing cache (in case of a failure, etc) while new workflows create a new session.
```